### PR TITLE
added infor in index.html file

### PR DESCRIPTION
**Pull request by MJWolberg89**
Lesson-3-1
Added information for index.html: Name, About, Experiences and Contact info.

### CODE REVIEW

**Comment by MJWolberg89** [comment]
Lesson 3-1 
Added information for index.html

**Comment by p34c310v3** [comment]
Looks good to me! :+1: :partying_face: :raised_hands: 

Just something to be aware of - about using your email address and phone number (if those are your real ones), since this is public and viewable by anyone. :grin: :smiley:

**Comment by MJWolberg89** [comment]
I wasn’t aware my phone # was on my GitHub?

-Micael

File: instructions/section-3/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+"About"
+I am 33 years old and live Bismarck, ND. I enjoy playing table top RPGs, Magic The Gathering and video games. I like to play music, skateboarding and moantain biking. Generally anything outdoors.
+
+"Experince"
+My general experince with work is cooking in the food industry but I have had some jobs in construction welding, plumbing and industrail painting. I just recently finished a seven month boot camp through The Last Mile that introduced me to computer programming through front-end developement. 
+
+"Connect"
+Email: mwolberg89@gmail.com
+Phone: (701)934-5964
```
Comment (p34c310v3) [inline instructions/section-3/index.html:11]: Btw, here's where I noticed your contact info (re: phone number). :slightly_smiling_face:

**Pull request merged**